qiime/beta_diversity.py

```diff
@@ -34,7 +34,7 @@
 import warnings
 warnings.filterwarnings('ignore', 'Not using MPI as mpi4py not found')
 
-from numpy import asarray
+from numpy import vstack
```

Comment:
no strong feelings here, can do if you'd like but the iteration with jenkins is large